packages/account-v2/src/pages/TwoFactorAuthentication/TwoFactorAuthenticationArticle.tsx

```diff
@@ -0,0 +1,21 @@
+import React from 'react';
+import classNames from 'classnames';
+import { SideNote, Text, useDevice } from '@deriv-com/ui';
+
+export const TwoFactorAuthenticationArticle = () => {
+    const { isTablet } = useDevice();
+    return (
+        <SideNote
+            className={classNames('py-[16px] px-[24px] h-fit w-[256px] sm:w-[calc(100%-32px)] sm:m-[16px]', {
```

Comment:
Isn't w-[256px] same as sm:w-[256px] ? Tailwind by default used small screen.So here may be if u meant the 256px width for non mobile screens shouldn't we use `md:` ?